neo/Network/P2P/LocalNode.cs

```diff
@@ -173,6 +209,23 @@ private void OnSendDirectly(IInventory inventory)
             Connections.Tell(inventory);
         }
 
+        private void OnDisconnectPayload(DisconnectPayload payload)
+        {
+            switch (payload.Reason)
+            {
+                case DisconnectReason.MaxConnectionReached:
+                case DisconnectReason.MaxPerAddressConnectionReached:
```

Comment:
`MaxConnectionPerAddressReached`

Reply:
Ok，I'll correct it.

Reply:
Thanks, it is more like a personal option, but I think it will look better in terms of comprehension.

Reply:
done